### PR TITLE
Restructured reduced schema with meaningful nodes

### PR DESCRIPTION
**Pull request by VisLab**
This is a first pass at restructuring with naming that has no conflicts. It will form a basis for development. Still many sections need work.

### CODE REVIEW

File: HED-schema-reduced.mediawiki [hunk 1]
```diff
@@ -96,60 +95,69 @@ There are four ways to define the scope of an event.  The most common method is
...
-**** Brake
-**** Stop
-**** Engage
-***** Turn signal
-***** Headlights
-***** Horn
-***** Radio
-**** Teleoperate <nowiki>[Control an object that you are not aboard]</nowiki> 
-** Ocular
+* Move
```
Comment (dungscout96) [inline HED-schema-reduced.mediawiki:103]: The word "Move" by itself has many meanings, including non-physical one such as "to influence". Should we use "Locomote" instead?

File: HED-schema-reduced.mediawiki [hunk 2]
```diff
@@ -96,60 +95,69 @@ There are four ways to define the scope of an event.  The most common method is
...
+** Breathe
+*** Inhale <nowiki>[A deliberate inbreath, e.g. in response to a task instruction. If incidental, use Action/Movement/Incidental/Inspiration]</nowiki>
+*** Exhale <nowiki>[A deliberate outbreath, e.g. in response to a task instruction. If incidental, use Action/Movement/Incidental/Expiration]</nowiki>
```
Comment (dungscout96) [inline HED-schema-reduced.mediawiki:106]: We can get rid of the example as Incidental will be made an Attribute

File: HED-schema-reduced.mediawiki [hunk 3]
```diff
@@ -96,60 +95,69 @@ There are four ways to define the scope of an event.  The most common method is
...
-**** Stop
-**** Engage
-***** Turn signal
-***** Headlights
-***** Horn
-***** Radio
-**** Teleoperate <nowiki>[Control an object that you are not aboard]</nowiki> 
-** Ocular
+* Move
+** Breathe
```
Comment (dungscout96) [inline HED-schema-reduced.mediawiki:104]: Should we put "Move chest" instead? And "Breathe" as a child of it?

File: HED-schema-reduced.mediawiki (hunk 1 above)
Comment (VisLab) [reply HED-schema-reduced.mediawiki:103]: Yes, better.

File: HED-schema-reduced.mediawiki [hunk 4]
```diff
@@ -158,47 +166,32 @@ There are four ways to define the scope of an event.  The most common method is
...
-** Musical
+** Communicate musically
 *** Hum 
 *** Whistle
 *** Vocalize <nowiki>[Make a sustained vowel sound such as, Ahhh]</nowiki>
 *** Sing
 *** Perform
 
-* Mental
+* Think
```
Comment (dungscout96) [inline HED-schema-reduced.mediawiki:193]: My two cents is that "Think" is not quite general enough. How about "Cerebrate"?...

File: HED-schema-reduced.mediawiki (hunk 2 above)
Comment (VisLab) [reply HED-schema-reduced.mediawiki]: Yes

File: HED-schema-reduced.mediawiki (hunk 3 above)
Comment (VisLab) [reply HED-schema-reduced.mediawiki]: I don't think so.  Breathe is the process of respiration.  I am taking it out of Locomote and put by itself.

File: HED-schema-reduced.mediawiki (hunk 4 above)
Comment (VisLab) [reply HED-schema-reduced.mediawiki:193]: It's a good word, but I had to look it up. It looks too much like celebrate.  Alternatives are Deliberate, Cogitate, Contemplate, Ponder, Mull over  (from the thesaurus).

Comment (dungscout96) [reply HED-schema-reduced.mediawiki:193]: Maybe Deliberate... All the other words seem too specific (hinting at intense and long thinking)

**Pull request merged**